src/library/scala/collection/mutable/OpenHashMap.scala

```diff
@@ -136,7 +149,11 @@ extends AbstractMap[Key, Value]
       None
     } else {
       val res = entry.value
```

Comment:
> the `entry.key` should also be updated at this point

That looks like it'd be practically a no-op. If `entry != null`, then `entry.key == key`. So maybe `entry.key ne key`, but why should this class care?

> It's somewhat contentious

Sounds like there's some history to this debate.